travvy/src/components/Landing.vue

```diff
@@ -4,8 +4,16 @@
     <img src="../images/map.jpg" alt="Picture of Map"/>
     <br>
     <br>
+    <!-- Button to go to the log inpage --> 
+    <router-link to="/Home" tag="button" class="button"><span>Log In</span></router-link>
+    <br>
```

Comment:
This looks good for now - it will need to be changed to take the user to the actual log in page after its created